flycheck.el

```diff
@@ -8022,6 +8040,10 @@ Requires Flake8 3.0 or newer. See URL
             (id (one-or-more (any alpha)) (one-or-more digit)) " "
             (message (one-or-more not-newline))
             line-end))
+  :enabled (lambda () (and flycheck-flake8-python-executable
```

Comment:
Is this a new variable that's not in this branch?  Or did you mean `flycheck-python-flake8-executable`?  The latter is already checked by Flycheck.